core/tst/software/aws/toolkits/core/utils/ZipUtilsTest.kt

```diff
@@ -62,9 +61,9 @@ class ZipUtilsTest {
         ZipFile(zipFile.toFile()).use { actualZip ->
             val actualEntry = actualZip.entries().toList().find { it.name == "file.txt" }
 
-            assert(actualEntry).isNotNull()
+            assertThat(actualEntry).isNotNull
```

Comment:
isNotNull `()`?

Reply:
Kotlin sees it as a property not a method

Reply:
weird but ok

Reply:
It is because it matches boolean Java Bean property syntax for a boolean